playbooks/roles/os_temps/tasks/setup_containers.yml

```diff
@@ -59,16 +71,16 @@
 # Setup project_repo templates and store variables for each template
 - name: "Setup project_repo templates and store variables for each template"
   include_tasks: "setup_os_templates.yml template_name={{ item.path }}.processed"
-  with_items: "{{ os_templates.files }}"
-  when: os_templates.files != ""
+  with_items: "{{ os_template_files }}"
+  when: os_template_files != ""
 
 # Handle loading of project_repo templates into OpenShift, retry 4 times
 - name: "Handle loading of project_repo templates into OpenShift"
   include_tasks: "handle_os_templates.yml"
   with_sequence: count=4
   loop_control:
     loop_var: attempt_number
-  when: os_templates.files != ""
+  when: os_template_files != ""
```

Comment:
I guess we could probably do- when: not os_template_files, but if it works then we can let it be.

Reply:
Yeah, that part has been there for some time, so I just changed the variable.